### PR TITLE
Add TYPO3 13 compatibility

### PR DESCRIPTION
**Pull request by mediaessenz**
#57

### CODE REVIEW

File: Classes/Middleware/JumpUrlHandler.php [hunk 1]
```diff
@@ -94,6 +98,30 @@ protected function redirectToJumpUrl(string $jumpUrl): ResponseInterface
...
 
+    /**
+     * @param ServerRequestInterface $request
+     * @return array
+     * @throws \JsonException
+     * @throws \TYPO3\CMS\Core\Cache\Exception\NoSuchCacheException
+     */
+    protected function getPageTsConfig(ServerRequestInterface $request): array
+    {
+        $pageInformation = $request->getAttribute('frontend.page.information');
```
Comment (bmack) [inline Classes/Middleware/JumpUrlHandler.php:109]: This change only works in v13, but the changes you make in composer.json says that the code should also stay compatible with older versions.

How about we make it work for v12 and v13 at the same time and drop the older versions?

Comment (mediaessenz) [reply Classes/Middleware/JumpUrlHandler.php:109]: Good idea.

Comment (mediaessenz) [reply Classes/Middleware/JumpUrlHandler.php:109]: Here you go ...

**Review by bmack — APPROVED** [review]

**Comment by bmack** [comment]
thanks for your work on this @mediaessenz

**Pull request merged**